server/src/services/entities/offer.ts

```diff
@@ -0,0 +1,65 @@
+import { env } from '../../libs/environment/env'
+
+import { EntityType, TwitterCard } from './types'
+
+const { DEEPLINK_PROTOCOL } = env
+
+export const OFFER: EntityType = {
+  API_MODEL_NAME: 'offer',
+  METAS_CONFIG: {
+    title(entity: Record<string, unknown>) {
+      return entity.name as string
```

Comment:
généralement, j'essaie d'éviter les `valeur as type` surtout dans le code de production (dans les tests, parfois, ça se comprends)

utiliser `valeur as type` dans le code de prod' est souvent un signe que les types pourrait etre mieux utiliser

si on sait que `entity` contient une propriété `name` qui est une `string`, peut etre qu'un `Record<string, unknown>` n'est pas adapté

que penserais-tu de faire un type spécifique en input ?

```suggestion
    title(entity: Entity) {
      return entity.name
```

ou

```suggestion
    title(entity: Offer) {
      return entity.name
```

Reply:
je trouve que c'est overkill (lire un schema swagger, généré des entités), c'est pourquoi je ne l'ai pas fait et ai toléré des données non setter. Dans l'idéal je suis d'accord, mais dans ce cas il faudrait faire un générateur comme ce qui a été fais dans ce qui génère notre api client et cela peut être fais en amélioration le ticket étant déjà assez conséquent et ceci retardant les tests et n'apportant que des verrous de typing.